src/common/json.cc

```diff
@@ -57,12 +63,12 @@ void JsonWriter::Visit(JsonObject const* obj) {
 }
 
 void JsonWriter::Visit(JsonNumber const* num) {
-  char number[NumericLimits<float>::kToCharsSize];
-  auto res = to_chars(number, number + sizeof(number), num->GetNumber());
+  std::array<char, NumericLimits<float>::kToCharsSize> number;
+  auto res = to_chars(number.data(), number.data() + number.size(), num->GetNumber());
```

Comment:
Is it possible to use `std::to_chars`, now that we are using C++17? Not blocking this release, let's consider using `std::to_chars` in the future.

Reply:
The implementation is unspecified by the standard. MSVC can produce different result than GCC.

Reply:
Even for integers?

Reply:
Not for interger. I will do some cleanups after 2.0